README.md

```diff
@@ -1,6 +1,59 @@
-# lit-element
+# LitElement
 
-## Base class for creating custom elements using Polymer and lit-html.
+## A simple base class for creating custom elements rendered with lit-html.
+
+LitElement uses [lit-html](https://github.com/Polymer/lit-html) to render into the
+element's Shadow DOM and [Polymer's](https://github.com/Polymer/polymer)
+PropertiesMixin to help manage element properties and attributes.
+
+LitElement uses a functional reactive pattern where any changes to element
+properties are batched and then rendered asynchronously. Users describe
+the entire rendering state declaratively, and lit-html ensures that updates
+are fast by creating the static DOM once and smartly updating only the parts of
+the DOM that change.
```

Comment:
Readme comments:
 * would be great to put a minimal code snippet which demonstrates the useful of utilising this base class.
 * Pull this paragraph out into highlighted dot points - 
   * **Batched rendering** - LitElement watches changes to properties and asynchronously renders these batched changes to reduce rendering overheads, 
   * **Declarative state**
  etc. with extended descriptions for each.
 * "functional reactive"? Sounds scary.